agent/handlers/v2/response_test.go

```diff
@@ -123,7 +123,7 @@ func TestTaskResponse(t *testing.T) {
 		state.EXPECT().ContainerMapByArn(taskARN).Return(containerNameToDockerContainer, true),
 	)
 
-	taskResponse, err := NewTaskResponse(taskARN, state, ecsClient, cluster, availabilityZone, containerInstanceArn, false)
+	taskResponse, err := NewTaskResponse(taskARN, state, ecsClient, cluster, availabilityZone, containerInstanceArn, false, false)
```

Comment:
No tests for `true` path?

Reply:
Good catch, that should be tested in the v4 package, will look into adding tests there.

Reply:
this is now addressed in TestTaskResponseWithV4Metadata and TestTaskResponse unit tests in v2 package.